zebra/rt_netlink.c

```diff
@@ -745,9 +745,7 @@ int netlink_route_change(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 		return 0;
 	}
 
-	if (!(rtm->rtm_family == AF_INET || rtm->rtm_family == AF_INET6
-	      || rtm->rtm_family == AF_ETHERNET
-	      || rtm->rtm_family == AF_MPLS)) {
+	if (!(rtm->rtm_family == AF_INET || rtm->rtm_family == AF_INET6)) {
```

Comment:
I am not sure it is necessary. ( I think of MPLS case).
what if someone wants to enhance the MPLS discovery ( ability for zebra to learn already MPLS entries in the kernel) ?

Reply:
Since we were already silently suppressing it below, I felt it best to consolidate them for the time being and just mark it as invalid.

Reply:
@pguibert6WIND if someone wants to enhance AF_MPLS then they can fix this if statement, until then the code has a hard check here for it, intentionally introduced by @rwestphal  and myself about a year ago to make sure we didn't accidently cause issues.